demos/README.md

```diff
@@ -45,3 +45,4 @@ Nimgame 2 Demos
 
 * [demo22](demo22) MpegGraphic
 
+* [demo23](demo23) PolygonGraphic
```

Comment:
Considering there are no actual `PolygonGraphic` type in the code, could you change the title (maybe for "Transform", etc.)?